### ISSUE TITLE
Chinese localization problem

### ISSUE THREAD

**Issue by ensigma96** [stom79/mastalab#362]
I've been using mastalab with my phone's system locale set to Simplified Chinese (`zh-CN`), and I feel that the translations are really far from good quality. Hoping to improve that, I only see Traditional Chinese (`zh-TW`) on Crowdin, but no Simplified Chinese.

Later I found out, more confusingly, that the "Traditional Chinese" translations on Crowdin are actually the source for [app/src/main/res/values-zh/strings.xml](https://github.com/stom79/mastalab/blob/develop/app/src/main/res/values-zh/strings.xml), which is exactly what i've been seeing under `zh-CN` locale. Indeed, it consists of (mostly) Simplified Chinese, though the contributors on Crowdin are gradually modifying it into Traditional Chinese. And if I change my locale to Traditional Chinese (`zh-TW`), the app shows untranslated English strings!

To sum up, the current problem is: there is only one `values-zh/strings.xml`, which consists of badly translated Simplified Chinese, and is actually providing localization for Simplified Chinese (`zh-CN`), but is receiving changes from Traditional Chinese (`zh-TW`) translations on Crowdin.

---
Mastalab version: 1.8.4

**Comment by stom79** [comment]
Thanks. The main problem is due to the synchronisation of Crowdin with the translation branch. These two versions keep their time to override each other due to the current settings (limited to 2 chars). I can't change these settings easily without breaking all other translations. 
So I will remove them from Crowdin and I will add Chinese Simplified (zh-CN) and traditional (zh) manually so that they can be edited on Github directly.

**Comment by ensigma96** [comment]
Thanks for that. But maybe it is possible to define some custom language mappings? I just searched around and found the [configuration file](https://github.com/openfoodfacts/openfoodfacts-androidapp/blob/master/crowdin.yml) of some project that uses Crowdin and has both zh-CN and zh-TW translations, not sure if that helps.

**Comment by stom79** [comment]
Thanks. I have updated the [config file](https://github.com/stom79/mastalab/blob/develop/crowdin.yml).
It seems to work like expected!
